docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.6
+
+RUN apt-get update
+RUN apt-get install -y ffmpeg mediainfo
```

Comment:
Please concatenate all `apt-get` commands in one line, as explained in the APT-GET subsection of [Dockerfiles best practices documentation](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/). You can have cache issues otherwise. Optionally, you can clear the apt cache after.

```dockerfile
RUN apt-get update && apt-get install -y \
    ffmpeg \
    mediainfo \
  && rm -rf /var/lib/apt/lists/*
```

By the way, you do not need mediainfo if you use ffmpeg for extracting metadata.